pyproject.toml

```diff
@@ -55,6 +55,7 @@ strict-rfc3339 = "^0.7"
 tabulate = "^0.9.0"
 uritemplate = "^4.1.1"
 adrf = "^0.1.1"
+xmltodict="0.13.0"
```

Comment:
was this added by hand? you're missing a corresponding update to the lockfile which will break builds

Reply:
`poetry add` works better for adding new dependencies

Reply:
Pushed lock file